sdk/spring/azure-spring-data-cosmos/src/main/java/com/azure/spring/data/cosmos/exception/CosmosNonStreamingOrderByBadRequestException.java

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.spring.data.cosmos.exception;
+
+import org.springframework.lang.Nullable;
+
+public class CosmosNonStreamingOrderByBadRequestException extends CosmosAccessException {
+
+    /**
+     * Construct a {@code CosmosDBAccessException} with the specified detail message.
+     * and nested exception.
+     * @param msg the detail message
+     * @param cause the nested Throwable
+     */
+    public CosmosNonStreamingOrderByBadRequestException(@Nullable String msg, @Nullable Throwable cause) {
```

Comment:
Do we really need this or is ComsosBadRequestException goo d enough?

Reply:
@kushagraThapar thoughts?